demo/admin/src/common/brevoModuleConfig/brevoContactsPageAttributesConfig.tsx

```diff
@@ -70,9 +70,7 @@ export const getBrevoContactConfig = (
         name: string;
     };
     input2State: (values?: AdditionalFormConfigInputProps) => {
-        email: string;
-        redirectionUrl: string;
-        attributes: { BRANCH?: Array<GQLBrevoContactBranch>; SALUTATION?: GQLBrevoContactSalutation; FIRSTNAME?: string; LASTNAME?: string };
+        attributes: { SALUTATION?: GQLBrevoContactSalutation; FIRSTNAME?: string; LASTNAME?: string };
```

Comment:
Why did the branch get removed here?

Reply:
c9a0e639675b628a572cb301f2df744e1cee88d8